doc/examples/segmentation/plot_mask_slic.py

```diff
@@ -52,18 +57,18 @@
 ax1, ax2, ax3, ax4 = ax_arr.ravel()
 
 ax1.imshow(img)
-ax1.set_title("Origin image")
+ax1.set_title('Original image')
 
-ax2.imshow(mask, cmap="gray")
-ax2.set_title("Mask")
+ax2.imshow(mask, cmap='gray')
+ax2.set_title('Mask')
 
 ax3.imshow(segmentation.mark_boundaries(img, slic))
 ax3.contour(mask, colors='red', linewidths=1)
 ax3.set_title("SLIC")
```

Comment:
Oops, I didn't catch this one. Anyhow.

Reply:
Yes, :slightly_smiling_face:. For consistency, please apply the same changes here.

Reply:
Sure. The reference here being https://github.com/scikit-image/scikit-image/pull/4850#discussion_r462005433 :wink: